jupyterhub/tests/test_services_auth.py

```diff
@@ -91,7 +91,7 @@ async def test_hubauth_token(app, mockservice_url, create_user_with_scopes):
         public_url(app, mockservice_url) + '/whoami/?token=%s' % token,
         allow_redirects=False,
     )
-    assert r.status_code == 302
+    assert r.status_code == 403
     assert 'Location' in r.headers
```

Comment:
This and the following lines need to be removed since it's no longer checking for a redirect